docs/lib/auth/fragments/native_common/authentication/signin/common.md

```diff
@@ -1,6 +1,41 @@
 
-<inline-fragment platform="ios" src="~/lib/auth/fragments/native_common/authentication/register_user/common.md"></inline-fragment>
-<inline-fragment platform="android" src="~/lib/auth/fragments/native_common/authentication/register_user/common.md"></inline-fragment>
+Auth category can be used to register a user into the auth plugin and sign-in the registered user using username and password in your app. AWS Cognito Auth Plugin is set up with AWS Cognito User Pools that manages the users and their properties.
 
-<inline-fragment platform="ios" src="~/lib/auth/fragments/native_common/authentication/common.md"></inline-fragment>
-<inline-fragment platform="android" src="~/lib/auth/fragments/native_common/authentication/common.md"></inline-fragment>
+## Prerequisites
+* An iOS application targeting at least iOS 11.0
+* Integrated with Amplify Auth and ability to sign in with an account
+  * For a full example, see: [Auth Getting Started](~/lib/auth/getting-started.md)
+
+## Configure the Backend
+
+The backend will be already configured if you have followed the basic setup as mentioned in the prerequisites.
+
+## Register a user
+
+A user should exists in the backend to move forward with the next step of authentication. The defaul CLI flow require a username, password and a valid email id as parameters to register a user. Create a UI to get these values from the user. When you have the needed details invoke the following function to initiate a signUp flow.
```

Comment:
Not sure what the first sentence means: suggest deleting it.

Reply:
We should have a defined CLI flow somewhere, and walk the user through it. "The default CLI flow" means nothing to a reader.